libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -38,6 +38,7 @@ uint32_t GCS_MAVLINK::reserve_param_space_start_ms;
 
 AP_HAL::Util::perf_counter_t GCS_MAVLINK::_perf_packet;
 AP_HAL::Util::perf_counter_t GCS_MAVLINK::_perf_update;
+bool GCS_MAVLINK::_in_update = false;
```

Comment:
Why? There's no multithread involved in this function. If it's called by another thread we already lost.